.github/workflows/ci-cd.yml

```
@@ -213,7 +213,7 @@ jobs:
         PIP_USER: 1
       run: >-
         PATH="${HOME}/Library/Python/3.11/bin:${HOME}/.local/bin:${PATH}"
-        pytest
+        pytest -m internal
```

```suggestion
        pytest
```

I think this is not needed

---

It is while 'not internal' is in the default arguments. If nobody has a better idea, then we'll probably have to skip disabling it by default and just recommend packagers to explicitly disable it in their test runs.

---

I don't see a way to override `not internal`

---

I wouldn't add it to the default arguments — the downstreams can read changelog and add markers.